srcmedia/ts/membership-graphs.ts

```diff
@@ -0,0 +1,236 @@
+import * as d3 from "d3"
```

Comment:
hopefully at some point we can move this to use a more typescript/es6 style where we just import the parts we need - keeps bundle size down and keeps stuff out of the global namespace

Reply:
yes, I wondered about that